### ISSUE TITLE
function "get_anlz_factor" return error

### ISSUE THREAD

**Issue by xx565763914** [bsolomon1124/pyfinance#25]
description:
   with pandas version: 1.3.4，there is a error when calling get_anlz_factor，which result in calling error when invoke function anlzd_stdev and sharpe_ratio

problem reason:
   at function get_anlz_factor，the map PERIODS_PER_YEAR doesn't has correct key type int，it's FreqGroup.FR_ANN at pandas 1.3.4

possible solution:
  use FreqGroup.value to get correction key.
![image](https://user-images.githubusercontent.com/25703389/190569422-a7297107-2cee-4bae-8720-9ced376cf071.png)

**Comment by bsolomon1124** [comment]
Good catch — your analysis was exactly right. In modern pandas `FreqGroup` is a plain `Enum` (not `IntEnum`), so the dict keyed on enum members couldn't be reached via the raw integer. Fixed in **v2.0.1** by keying `PERIODS_PER_YEAR` on `FreqGroup.<X>.value` as you suggested.

Release: <https://pypi.org/project/pyfinance/2.0.1/>

Thanks for the detailed report and the screenshot — it made the fix trivial. Closing.
